File: abc/abc165/a.py
# https://atcoder.jp/contests/abc165/submissions/34774162

k = int(input())
a, b = map(int, input().split())
print("OK" if (b // k - (a - 1) // k) > 0 else "NG")
#print((b / k - (a - 1) / k))

File: abc/abc165/a.cpp
// https://atcoder.jp/contests/abc165/submissions/26203264

#include <bits/stdc++.h>
using namespace std;
int main() {
  int k, a, b;
  cin >> k >> a >> b;
  if(a <= (b / k * k)) cout << "OK";
  else cout << "NG";
}
